src/pages/editor/PublicationChannelTableRow.tsx

```diff
@@ -0,0 +1,37 @@
+import { Chip, TableCell, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { useFetchPublisher } from '../../api/hooks/useFetchPublisher';
+import { ClaimedChannel } from '../../types/customerInstitution.types';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { getLanguageString } from '../../utils/translation-helpers';
+
+interface ChannelClaimTableRowProps {
+  channel: ClaimedChannel;
```

Comment:
Kanskje bruke samme navn på prop også, siden `channel` (fra kanalregister) og `ClaimedChannel` (blanding av kanal og kunde i NVA) er to ulike ting
```suggestion
  claimedChannel: ClaimedChannel;
```